core/internal/data_types/data_types.go

```diff
@@ -111,11 +111,18 @@ func resolveTypes(data interface{}, invalid ...bool) TypeRepresentation {
 				Length: len(v),
 			},
 		}
-	case int, float64:
+	case int, int8, int16, int32, int64:
+		return TypeRepresentation{
+			Name: NumberTypeName,
+		}
+	case uint, uint8, uint16, uint32, uint64:
+		return TypeRepresentation{
+			Name: NumberTypeName,
+		}
+	case float32, float64:
```

Comment:
had to fix this as launch test was failing, with Unknown type instead of Number. 
I think this was triggered because we use the other json pkg, so types were a bit different,
I just added more number types